punishments.js

```diff
@@ -1032,12 +1032,12 @@ Punishments.getRoomPunishments = function (user, publicOnly) {
  * @param {User} user
  */
 Punishments.monitorRoomPunishments = function (user) {
-	let num = (Config.monitorminpunishments !== undefined ? Config.monitorminpunishments : 3); // Default to 3 if the config option isn't present.
-	if (!num) return;
+	const MINIMUM_PUNISHMENTS_TRIGGER = (typeof Config.monitorminpunishments === 'number' ? Config.monitorminpunishments : 3); // Default to 3 if the Config option is not defined or valid
```

Comment:
We do not use allcaps/underscores for constants unless they're actually constants, not config values.